bindings/node/lib/common.js

```diff
@@ -1,69 +1,71 @@
 'use strict';
 
-function debug(msg) {
-  if (process.env.MONGODB_CRYPT_DEBUG) {
-    console.log(msg);
+module.exports = function() {
```

Comment:
I think its probably fine to skip the dependency injection for modules that don't need it, or which are exclusively internal modules (like this one)

Reply:
yeah, I'm okay with getting rid of it for `common` and `mongocryptdManager`. `stateMachine` is still going to need it to get the `MongoTimeoutError`